ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -729,6 +729,7 @@ ethernet_interfaces:
     isis_passive: < boolean >
     isis_metric: < integer >
     isis_network_point_to_point: < boolean >
+    isis_circuit_type: < level-1-2 | level-1 | level-2 >
```

Comment:
please add port-channel interface support. It is already covered by the documentation template, so should be simple to add. Thanks